### PR TITLE
jack 1.9.22

### PR DESCRIPTION
**Pull request by p-linnane**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by equal-l2** [comment]
The binaries used in the test (e.g. `jack_wait`, `jack_midiseq`, ...) have decoupled from `jack2` and now live in a separate package (https://github.com/jackaudio/jack-example-tools).
You need to either build them with `jack2` or modify the test not to use them.

**Comment by chenrui333** [comment]
> The binaries used in the test (e.g. `jack_wait`, `jack_midiseq`, ...) have decoupled from `jack2` and now live in a separate package (https://github.com/jackaudio/jack-example-tools). You need to either build them with `jack2` or modify the test not to use them.

yeah, we can build the example-tools as part of the formula itself, but I did see the example-tools also depend on jack1. I will check a bit tomorrow.

**Comment by chenrui333** [comment]
We can create a separate formula for `jack-example-tools`

File: Formula/jack.rb [hunk 1]
```diff
@@ -25,11 +25,9 @@ class Jack < Formula
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
-  depends_on "python@3.10" => :build
+  depends_on "python@3.11" => :build
   depends_on "berkeley-db"
   depends_on "libsamplerate"
-  depends_on "libsndfile"
-  depends_on "readline"
```
Comment (chenrui333) [inline Formula/jack.rb:32]: these deps are for `jack-example-tools`

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**